src/routes/admin/mentee/mentee.route.ts

```diff
@@ -1,9 +1,11 @@
 import express from 'express'
 import { requireAuth } from '../../../controllers/auth.controller'
 import { getAllMenteeEmails } from '../../../controllers/admin/email.controller'
+import { updateMenteeStatus } from '../../../controllers/admin/mentee.controller'
 
 const menteeRouter = express.Router()
 
 menteeRouter.get('/emails/', requireAuth, getAllMenteeEmails)
+menteeRouter.put('/:menteeId/status/', requireAuth, updateMenteeStatus)
```

Comment:
Can you update the mentor route as well?  This will be used by mentors to approve/reject mentees as well @kumuditha-udayanga